wrangler-core/src/test/java/co/cask/directives/transformation/ParseDateTest.java

```diff
@@ -121,7 +121,7 @@ public void testSimpleDateWithPatterns() throws Exception {
 
     List<Row> rows = TestingRig.execute(directives, Arrays.asList(row1));
     LocalDate localDate = LocalDate.of(2016, 12, 10);
-    LocalTime zeroTime = LocalTime.of(0, 0);
+    LocalTime zeroTime = LocalTime.of(8, 0);
```

Comment:
This is because `Date` object uses system default zone. https://github.com/data-integrations/wrangler/blob/develop/wrangler-core/src/main/java/co/cask/directives/parser/ParseSimpleDate.java#L86

Reply:
We can set timezone using `formatter.setTimeZone(TimeZone.getTimeZone("UTC"));`. but that would mean whenever there is no timezone specified in the data, the directive will assume it to be UTC. However, existing behavior considers date without timezone to be system default.

Reply:
It has to be consistent across the directives. This unit test will break on any system that is not in the PST timezone.

I think parse-as-simple-date should use UTC.

Reply:
For example, imagine if the output changed between pipeline runs because the cluster happened to be on the east coast for one run and the west coast for another.

Reply:
Right, we can use UTC.`parse-as-simple-date` directive does not have a way to provide timezone, may be we can add an optional parameter to that directive?

Reply:
yeah I think that makes sense